server/boot/z-customer-accounts.js

```diff
@@ -0,0 +1,24 @@
+var async = require('async');
+
+module.exports = function(app) {
```

Comment:
hey Simon, the async problem might be very complex, if the boot script has to be async then I am afraid I have to refactor the whole code, because now the endpoint urls in front-end and tests rely on the creation sequence of customers. 
I am not sure whether the customers are created in sequence if I use sync boot script, if yes, then this would better to remain same(actually this is the way that the code works in `loopback-example-embedded-relations`), if not, the whole example would be changed.
Thanks!